CMakeLists.txt

```diff
@@ -32,6 +32,17 @@ option(GINKGO_CUDA_DEFAULT_HOST_COMPILER "Tell Ginkgo to not automatically set t
 option(GINKGO_JACOBI_FULL_OPTIMIZATIONS "Use all the optimizations for the CUDA Jacobi algorithm" OFF)
 option(BUILD_SHARED_LIBS "Build shared (.so, .dylib, .dll) libraries" ON)
 
+if(BUILD_SHARED_LIBS AND WIN32 AND (GINKGO_BUILD_TESTS OR GINKGO_BUILD_EXAMPLES OR GINKGO_BUILD_BENCHMARKS))
+    # Change shared libraries output only if this build has executable program with shared libraries.
+    set(GINKGO_CHANGED_SHARED_LIBRARY TRUE)
+    option(GINKGO_CHECK_PATH "Tell Ginkgo to check environment PATH is available for this build." ON)
+    set(GINKGO_WINDOWS_SHARED_LIBRARY_RELPATH "windows_shared_library" CACHE STRING
+        "Set Ginkgo's shared library relative path in windows. This absoulte path must be in environment PATH. Current default is `windows_shared_library`.")
```

Comment:
Correction: `This absolute path must be in the environment variable PATH.`
Problem is, which absolute path? At first, you say this is a relative path.